src/ranged.cpp

```diff
@@ -161,19 +161,79 @@ bool player::handle_gun_damage( item &it, int shots_fired )
 {
     if( !it.is_gun() ) {
         debugmsg( "Tried to handle_gun_damage of a non-gun %s", it.tname() );
+        shots_fired = shots_fired + 0; // left this here in case someone needs shots_fired in the future.
         return false;
     }
-
+    int bpjamoccurredthisturn = 0;
```

Comment:
Your underscore key is broken. You should really fix that. Is's really difficult to write good variable names without it.